board.py

```diff
@@ -39,3 +39,51 @@ def __setitem__(self, pos: str, item: Piece):
         row = int(pos[1]) - 1
         column = self.columns[pos[0]]
         self.matrix[row][column] = item
+
+    def inside_board(self, i: int, j: int):
```

Comment:
```suggestion
    def inside_board(self, i: int, j: int) -> bool:
```
add type returns for the rest of the functions as well, even if it is `-> None`